views/compliment-show.hbs

```diff
@@ -0,0 +1 @@
+<h2> {{compliment}} </h2>
```

Comment:
I expect this isn't working, because you passed an object with a key of "compliments".